### ISSUE TITLE
Discernir usuário normal de usuário admin

### ISSUE THREAD

**Issue by guites** [guites/cinemaempoa#40]
Atualmente qualquer usuário logado no sistema pode cadastrar/editar sessões e importar arquivos json.

O ideal é que essas funções sejam só pra usuários de tipo administrador. Usuários normais, por enquanto, não podem fazer nada, mas essa distinção seria um início da direção de no futuro a pessoa poder logar, curtir filmes, avaliar, comentar, etc.

Talvez um campo `is_admin` na tabela de usuários já resolva, e filtrar as ações não só pra usuários logados, mas usuários logados que sejam admin.

**Comment by KozielGPC** [comment]
Bom dia @guites! Fiquei itneressado nessa issue aqui, posso pegar pra trabalhar nela?

**Comment by guites** [comment]
> Bom dia @guites! Fiquei itneressado nessa issue aqui, posso pegar pra trabalhar nela?

Claro!!! Essa implementação vai ser o ponto de partida pra conseguirmos adicionar umas coisas maneiras tipo permitir que a galera curta os filmes, proponha correções nas descrições, ajudem com a adição de texto descritivo nas fotos e tudo mais (claro que cada coisa vai ser feita num issue separado).

Qualquer dúvida é só falar.

**Comment by KozielGPC** [comment]
Certo, valeu! Fim de semana ja vou pegar pra mexer aqui

**Comment by KozielGPC** [comment]
@guites Finalizei o desenvolvimento dessa task e já abri o PR! Qualquer coisa estou a disposição
